src/Admin/entrypoint.sh

```diff
@@ -5,52 +5,27 @@
 GROUPNAME="bitwarden"
 USERNAME="bitwarden"
 
-CURRENTGID=`getent group $GROUPNAME | cut -d: -f3`
-LGID=${LOCAL_GID:-999}
+LUID=${LOCAL_UID:-0}
+LGID=${LOCAL_GID:-0}
 
-NOUSER=`id -u $USERNAME > /dev/null 2>&1; echo $?`
-LUID=${LOCAL_UID:-999}
+# Step down from host root to well-known nobody/nogroup user
 
-# Step down from host root
-
-if [ $LGID == 0 ]
+if [ $LUID -eq 0 ]
 then
-    LGID=999
+    LUID=65534
```

Comment:
So containers will never run as root anymore? Even if run from sudo?

Reply:
We do not want services to run as root, for security reasons.
`sudo` does nothing more than granting `root` rights, thus allowing calling user (which has sort of switched to `root`) to access docker subsystem (and then to start its containers).
Containers then start services using the UID/GID from the `uid.env` file.

On some systems, adding users to the `docker` group is another solution to allow them to use Docker, thus without needing `sudo` anymore : `usermod -aG docker myuser`